src/proj_api.h

```diff
@@ -31,16 +31,16 @@
   * This version number should be updated with every release!  The format of
   * PJ_VERSION is
   *
-  * * Before version 4.10.0: PJ_VERSION=MNP where M, N, and P are the major,
+  * * Before version 5.0.0: PJ_VERSION=MNP where M, N, and P are the major,
   *   minor, and patch numbers; e.g., PJ_VERSION=493 for version 4.9.3.
   *
-  * * Version 4.10.0 and later: PJ_VERSION=MMMNNNPP later where MMM, NNN, PP
+  * * Version 5.0.0 and later: PJ_VERSION=MMMNNNPP later where MMM, NNN, PP
```

Comment:
If 5.0.0 is to be PJ_VERSION=50000,  then please change this to read as so:

`* * Version 5.0.0 and later: PJ_VERSION=MMNNPP later where MM, NN, PP`

Reply:
Although not very obvious, and probably incorrect, I was trying to follow the recipe in the comment. It says that minor and patch numbers should be padded with zeros. So what I've got here is 5 *padding zero* 0 *padding zero* 0. I'm happy to change it if it's too obscure.